packages/SystemsThinkingWorkbench-Core.package/STWConnection.class/class/new.st

```diff
@@ -0,0 +1,3 @@
+instance creation
+new
+	^ self from: 0@0 to: 50@50 color: Color black width: 2
```

Comment:
yes, it's magic. but without it, arrows don't work, at least for me. Points given are placeholders and should be overwritten once placed in the world.